src/test/java/com/rocketden/main/service/ProblemServiceTests.java

```diff
@@ -173,10 +173,10 @@ public void getProblemsSuccess() {
         problem.setName(NAME);
         problem.setDescription(DESCRIPTION);
         problem.setDifficulty(ProblemDifficulty.EASY);
-
+        problem.setApproval(true);
         List<Problem> expected = new ArrayList<>();
         expected.add(problem);
-
+        //do findAllByApproval once it works
```

Comment:
nit: remove comment (and the other new comments/commented out lines in the file)

Reply:
k